src/apps/inventory/views.py

```diff
@@ -127,17 +133,27 @@ class ProductCreateView(LoginRequiredMixin, UserPassesTestMixin, CreateView):
     model = Product
     form_class = ProductForm
     template_name = "product_create.html"
-    success_url = reverse_lazy(
-        "product_by_seller"
-    )  # Redirect to a success URL (change this to your desired URL)
+    success_url = reverse_lazy("product_by_seller")
 
     def form_valid(self, form):
         form.instance.seller = self.request.user
-        return super().form_valid(form)
+        response = super().form_valid(form)
+
+        images_form = ProductImageForm(self.request.POST, self.request.FILES)
+        if images_form.is_valid():
+            for image in self.request.FILES.getlist("image"):
+                ProductImage.objects.create(product=self.object, image=image)
```

Comment:
I am not sure, but maybe we can use bulk_create method in case when we have to create several pics ,to optimize requests to our DB

Reply:
Good suggestion, let's keep it as it is for now, because we don't have such scale)

Reply:
Good suggestion, let's keep it as it is for now, because we don't have such scale)

Reply:
Good suggestion, let's keep it as it is for now, because we don't have such scale)

Reply:
Can we save images form with .save() method? I think it should know how to save objects